src/main/webapp/style.css

```diff
@@ -52,21 +56,32 @@ input[type=submit] {
   text-transform: uppercase;
 }
 
+input[type=submit] {
+  background: #0dbd00;
+}
+
+button,
 input[type=submit]:hover {
-  background: #0ba600;
+  filter: brightness(90%);
   transition: all .4s ease-in-out;
 }
 
+button,
 input[type=submit]:focus {
   outline: 0;
 }
 
+button {
+  background: blue;
```

Comment:
Nit: I don't love this shade. To my eyes, it seems too bright for the relatively large size of the button. I'd prefer either something a bit darker, or to flip to having the text be black and go with a much lighter blue.

Reply:
Toned down the blue and added dim on hover to all buttons. Let me know if you think it should be lighter/darker!

Reply:
I think it looks great! The blue theme is really nice

Reply:
Looks great, thank you!